xml/System.Runtime.CompilerServices/Unsafe.xml

```diff
@@ -594,7 +594,7 @@
         <param name="destination">The destination address to copy to.</param>
         <param name="source">The source address to copy from.</param>
         <param name="byteCount">The number of bytes to copy.</param>
-        <summary>Copies bytes from the source address to the destination address without assuming architecture dependent alignment of the addresses.</summary>
+        <summary>Copies bytes from the source address to the destination address. Unlike <xref:System.Runtime.CompilerServices.Unsafe.CopyBlock>, this will succeed even while we use the platform which requires alignment and any of the given location is unaligned.</summary>
```

Comment:
```suggestion
        <summary>Copies bytes from the source address to the destination address. Unlike <see cref="Overload:System.Runtime.CompilerServices.Unsafe.CopyBlock">, this write will succeed on platforms that require alignment even if the destination address is unaligned.</summary>
```

Reply:
I used the sentence "any of the given location" because it also takes source address and it could be also unaligned.
What do you think about it?

Any other suggestion seems to be really good! Thanks!